### DISCUSSION TITLE
移除ListDetailView

### DISCUSSION THREAD

**Discussion by Sanlorng** [MicaApps/Mail#30] [Ideas]
当前ListDetailView，不足以满足邮件列表管理（需要手动拿到ListView对象，然后进行选择模式的切换），多方式详情页显示（目前只能显示选中的列表，当存在如显示新建邮件页面时，则无法直接显示，需要通过创建一个对象，插入列表里并选中它才能显示。
可以考虑Fork ListDetailView的实现，并实现上述的功能。

**Comment by zhuxb711** [comment]
不是反对替换ListDetailsView哈。

提两点：
手动遍历VisualTree拿到内部对象是Uwp应用的基操无可避免。

微软自己的邮件也是会在新建邮件时插入一个列表项，只是会打上 草稿 的前缀，所以我并不觉得这个是什么阻碍性的问题
